handlers/watchers/__init__.py

```diff
@@ -0,0 +1,51 @@
+from eventhandler import EventHandler
+import ConfigParser
+import os
+
+WATCHERS_CONFIG_FILE = os.path.join(os.path.dirname(__file__), 'watchers.ini')
+
+def get_config():
+    config = ConfigParser.ConfigParser()
+    config.read(WATCHERS_CONFIG_FILE)
+    return config
+
+def build_message(mentions):
+    message = [ 'Watched files:' ]
+    for (watcher, file_names) in mentions.items():
+        message.append(" * @{}: {}".format(watcher, ', '.join(file_names)))
+
+    return '\n'.join(message)
+
+class WatchersHandler(EventHandler):
+    def on_pr_opened(self, api, payload):
+        diff = api.get_diff()
+        changed_files = []
+        for line in diff.split('\n'):
+            if line.startswith('diff --git'):
+                changed_files.extend(line.split('diff --git ')[-1].split(' '))
+
+        config = get_config()
+        mentions = {}
+
+        repo = '/'.join([api.owner, api.repo])
+        try:
+            watchers = config.items(repo)
+        except ConfigParser.NoSectionError:
+            return # No watchers
+
+        for (watcher, watched_files) in watchers:
+            watched_files = watched_files.split(' ')
+            for changed_file in changed_files:
+                for watched_file in watched_files:
+                    if watched_file in changed_file:
```

Comment:
I'm tempted to use `if changed_file.starts_with(watched_file)` instead to avoid false-positives. This will require stripping the `a/` and `b/` prefixes off of the filenames earlier when parsing the `diff --git` lines.